packages/open-next/src/build/installDeps.ts

```diff
@@ -23,11 +23,18 @@ export function installDependencies(
     logger.info(`Installing dependencies for ${name}...`);
     // We then need to run install in the tempDir
     // We don't install in the output dir directly because it could contain a package.json, and npm would then try to reinstall not complete deps from tracing the files
-    const installCommand = `npm install --arch=${
-      installOptions.arch ?? "arm64"
-    } --platform=linux --target=${installOptions.nodeVersion ?? "18"} --libc=${
-      installOptions.libc ?? "glibc"
-    } ${installOptions.packages.join(" ")}`;
+    const archOption = installOptions.arch
+      ? `--arch=${installOptions.arch}`
+      : "";
+    const targetOption = installOptions.nodeVersion
+      ? `--target=${installOptions.nodeVersion}`
+      : "";
+    const libcOption = installOptions.libc
+      ? `--libc=${installOptions.libc}`
+      : "";
+
+    const additionalArgs = installOptions.additionalArgs ?? "";
+    const installCommand = `npm install --platform=linux ${archOption} ${targetOption} ${libcOption} ${additionalArgs} ${installOptions.packages.join(" ")}`;
```

Comment:
what if the platform is `Windows`?

Reply:
People still use windows as a server ? :joy: 
That's only for the server. Honestly there is probably dozens of other issue trying to run OpenNext on windows.

Reply:
I read someone was insterested in running ON on Azure 🤷 
We'll cross that bridge when we get there.